client/src/components/datasetCard.module.scss

```diff
@@ -0,0 +1,40 @@
+.datasetCard {
+  background-color: white;
+  padding: 2.7rem 3rem 3rem 3rem;
```

Comment:
Good call on using `rem` - I was reading about this yesterday and I didn't realize the limitations around using `em`, which is local to its context. `rem` definitely seems superior

Reply:
yeah totally still up in the air on this.. also heard that rem works well!